origin.html

```diff
@@ -12,26 +12,85 @@
     <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet" />
     <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
 </head>
-
 <body>
     <header>
         <nav class="navbar">
             <a class="logo"><h3>Hector Andre Lau</h3></a>
-            <ul><li><img src="./images/Union.svg" alt="Menu"></li></ul>
+            <ul class="ulnavbar"><li><img src="./images/Union.svg" alt="Menu"></li></ul>
         </nav>
     </header>
     <section>
```

Comment:
[OPTIONAL] Great work on using semantic tag here :+1:. However, I think you misunderstood the comment by the previous code reviewer. That is why I want to make clear:

- [ ] When thinking about the HTML document flow, it is best to think that the main `container` or `parent  element` for the content is `<body/>`, it (`<body/>`) have three `child elements` - `<header />`, `<main />` and `<footer/>`. you did a great job, using your `<header/>` for your `toolbars` (logo, nav etc.), indeed it is meant for those purposes. Semantic tag `<main />` is where most of your main content goes in. `<main/>` can also have its own child elements and as per semantics, they are to be `<section/>` s. if the content for your `<section/>` can be divided into further categories or containers you can go a head and use `<div/>` s. If we visualize what we have, we will have  the document flow similar to the following: 
```HTML
<body>
  <header>
    <hgroup>
      <h1>
        ...
      </h1>
      <h2>
        ...
      </h2>
    </hgroup>
    <nav>
      <ul>
        ....
      </ul>
    </nav>
  </header>
  <main>
    <section>
      <div>
        ...
      </div>
    </section>

    <section>
      <div>
        ...
      </div>
    </section>
  </main>
  <footer>
    ...
  </footer>
</body>
```
Documenting your HTML in this way will make it well structured and easy to follow the logic. I hope it clarifies things.

In the case of your project, you are using `<section/>` instead of `<main/>` and `<div/>` instead of `<section/>`. It is a required change for this project, however, I am marking it as [OPTIONAL] because  I believe what you have done is enough to go to the next milestone. At the same time, please note, making these changes will help you in the further milestones and will help you to understand the core purpose of Semantic tags which are essential in improving the web page's `accessibility`